exhale/graph.py

```diff
@@ -1679,34 +1679,6 @@ def fileRefDiscovery(self):
         # IMPORTANT: do not set the parent field of anything being added as a child to the file
         #
 
-        # hack to make things work right on RTD
```

Comment:
Hmmm.  I guess this really isn't used?  It definitely used to be, I'll need to investigate this one more closely.  Originally without it `STRIP_FROM_PATH` from Doxygen didn't strip anything on RTD, producing absurdly long path names for things like file locations.

This may mean they possibly updated Doxygen?  In any case, if this is no longer necessary then I may be able to remove that from the required configuration entries!

Reply:
Actually now that you mention it I remember having an issue with STRIP_FROM_PATH in the first few runs with only the first fix regarding file handlers applied, and it has disappeared. I'll investigate if its disappearance is related to this chunk